docs/source/user/first-look-at-DP.ipynb

```diff
@@ -0,0 +1,649 @@
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>2251</th>\n",
+       "      <td>3</td>\n",
+       "      <td>27</td>\n",
+       "      <td>3</td>\n",
+       "      <td>2</td>\n",
+       "      <td>3</td>\n",
+       "      <td>8</td>\n",
+       "      <td>1</td>\n",
+       "      <td>26</td>\n",
+       "      <td>29</td>\n",
+       "      <td>38</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      sex  age  maritalStatus hasChildren  highestEducationLevel  \\\n",
+       "2251    3   27              3           2                      3   \n",
+       "\n",
+       "     sourceOfStress smoker optimism lifeSatisfaction selfEsteem  \n",
+       "2251              8      1       26               29         38  "
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df.loc[(df['sex'] == 3) & (df['age'] == 27)]\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In a dataset of 7,000 records, these two identifiers alone uniquely identified my coworker, allowing me to see their responses-- they are unmarried, but living with their partner. \n",
+    "In addition I can also see that they are a smoker.\n",
+    "\n",
+    "In other situations there are more nuanced techniques that can be used to determine if an individual exists in a dataset (called a membership attack)\n",
+    "or reconstruct parts of, or even the entire dataset, from summary statistics.\n",
+    "\n",
+    "Giving strong formal privacy guarantees requires the rigorous mathematical grounding that differential privacy provides."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## 1. Unit of Privacy\n",
+    "The first step of a DP analysis is to identify the unit of privacy.\n",
+    "Before we start to reason about privacy, we need to isolate just what we're trying to protect.\n",
+    "\n",
+    "In the teacher survey, each teacher contributes one row to the dataset, so our unit of privacy is a single row."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "max_contributions = 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "> Broadly speaking, differential privacy can be applied to any medium of data for which you can define a unit of privacy.\n",
+    "> In other contexts, the unit of privacy may be multiple rows, a user ID, or a node or edge in a graph.\n",
+    "\n",
+    "The unit of privacy frames the privacy guarantee provided by $\\epsilon$.\n",
+    "An analysis is only $\\epsilon$-DP if the methods can conceal any change made to the input dataset as coarse as the unit of privacy.\n",
```

Comment:
The usage of $\epsilon$ and $\epsilon$-DP is defined at the end of Section 3 (Construct Measurement), but I'm thinking it could be a good idea to also add excerpts of said usage here, as this section has the first reference of $\epsilon$.

```suggestion
    "The unit of privacy frames the privacy guarantee provided by $\\epsilon$, which is a commonly-used proxy to quantify the worst-case risk to any individual.\n",
    "It is customary to refer to a data release with such bounded risk as $\\epsilon$-DP and as such, an analysis is only $\\epsilon$-DP if the methods can conceal any change made to the input dataset as coarse as the unit of privacy.\n",
```

Reply:
I've restructured to add a section talking about the privacy parameters after the unit of privacy section. Thanks.